lldb/test/Shell/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 add_custom_target(lldb-shell-test-deps)
 set_target_properties(lldb-shell-test-deps PROPERTIES FOLDER "LLDB/Tests")
 add_dependencies(lldb-shell-test-deps lldb-test-depends)
+get_target_property(clang_resource_headers_dir clang-resource-headers INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
The `clang-resource-headers` target may not exist if you build LLDB standalone (that is, with a prebuilt LLVM/Clang). See the end of `cmake/modules/LLDBStandalone.cmake`